src/inspector/view/qml/MuseScore/Inspector/text/TextSettings.qml

```diff
@@ -225,6 +225,28 @@ Column {
 
     SeparatorLine { anchors.margins: -12 }
 
+    SpinBoxPropertyView {
+        id: textLineSpacingSection
+        anchors.left: parent.left
+        anchors.right: parent.horizontalCenter
+        anchors.rightMargin: 2
+
+        navigationName: "Line Spacing"
+        navigationPanel: root.navigationPanel
+        navigationRowStart: styleSection.navigationRowEnd + 1
+
+        titleText: qsTrc("inspector", "Line Spacing")
+        measureUnitsSymbol: qsTrc("global", "li")
+        propertyItem: root.model ? root.model.textLineSpacing : null
+
+        decimals: 2
+        step: 0.1
+        minValue: 0
+        maxValue: 99
+    }
+
+    SeparatorLine { anchors.margins: -12 }
+
     DropdownPropertyView {
         id: textStyleSection
         titleText: qsTrc("inspector", "Text style")
```

Comment:
Down here, you need to change
```qml
navigationRowStart: cornerRadiusSection.navigationRowEnd + 1
```
to
```qml
navigationRowStart: textLineSpacingSection.navigationRowEnd + 1
```

So, the navigation order would become `cornerRadiusSection` > `textLineSpacingSection` > `textStyleSection`. 

(GitHub doesn't let me attach a comment to that line, that's why I'm attaching it here)